packages/experiments/src/components/Button/Button.styles.ts

```diff
@@ -204,14 +204,15 @@ export const ButtonStyles: IButtonComponent['styles'] = (props, theme, tokens):
 
   const globalClassNames = getGlobalClassNames(
     {
-      icon: 'ms-Icon'
+      msButton: 'ms-Button',
+      msButtonIcon: 'ms-Button-Icon'
```

Comment:
I think BEM style (outside of Microsoft's component naming) should start lowercase.. looking at [existing Button snapshots](https://github.com/OfficeDev/office-ui-fabric-react/blob/97a5855cdb0f2c99269e6b435231577998bd1621/packages/office-ui-fabric-react/src/components/Button/__snapshots__/Button.test.tsx.snap#L72) seems to confirm that. I think this should be `ms-Button-icon` instead.